appveyor.yml

```diff
@@ -1,39 +1,61 @@
-version: 0.0.{build}
+version: 0.0.0+{build}
 
 skip_tags: true
 
 skip_branch_with_pr: true
 
-image: 
-- Visual Studio 2015
-- Visual Studio 2017
-- Visual Studio 2019
-- Ubuntu
-- macOS
-
 build:
   verbosity: minimal
 
-build_script:
-- pwsh: .\build.ps1 -Verbose
+test: off

```

Comment:
What does `test: off` mean?

Reply:
It turns off AppVeyor automatically finding and running tests in compiled .NET assemblies. We probably don't need to turn that off in this build.

Reply:
Removed.